### PR TITLE
Adding Numato Waxwing boards

### PR DESCRIPTION
**Pull request by FelixVi**
Board definition files and blinky example included

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -46,3 +46,5 @@ nosetests.xml
 /examples/boards/ppro/xilinx
 /examples/build/xilinx
 
+/SDRAM_Controller
```
Comment (cfelton) [inline .gitignore]: This isn't actually in the repo, doesn't make sense to add.

Comment (FelixVi) [inline .gitignore]: Sorry, that slipped in when I was working on the SDRAM controller...
It's removed now

File: .gitignore [hunk 2]
```diff
@@ -43,6 +43,11 @@ pip-log.txt
 nosetests.xml
 
 # generated examples
-/examples/boards/ppro/xilinx
-/examples/build/xilinx
+/examples/*/xilinx
+/examples/*/*/xilinx
```
Comment (cfelton) [inline .gitignore]: You shouldn't need /*/* a single /*/ should cover it, no?  (simple tests verify only /*/ is needed).

**Comment by FelixVi** [comment]
Changes got merged along with PR #37

**Pull request closed without merging**